### PR TITLE
Add description in readme and composer.json

### PR DESCRIPTION
**Pull request by samsonasik**
|    Q          |   A
|-------------- | ------
| Documentation | yes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://travis-ci.com/laminas/laminas-servicemanager-migration.svg?branch=master)](https://travis-ci.com/laminas/laminas-servicemanager-migration)
 [![Coverage Status](https://coveralls.io/repos/github/laminas/laminas-servicemanager-migration/badge.svg?branch=master)](https://coveralls.io/github/laminas/laminas-servicemanager-migration?branch=master)
 
-This library provides …
+Migrate to next laminas-servicemanager 4.0 compatible code utilize [`Rector`](https://github.com/rectorphp/rector) rules.
```
Comment (weierophinney) [inline README.md]: Maybe:

```markdown
Migrate your code to laminas-servicemanager 4.x compatibility utilizing [Rector](https://github.com/rectorphp/rector) rules.
```

File: composer.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "laminas/laminas-servicemanager-migration",
-    "description": "",
+    "description": "Migrate to next laminas-servicemanager 4.0 compatible code utilize Rector rules.",
```
Comment (weierophinney) [inline composer.json]: And mirror the changes here.

**Review by weierophinney — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (samsonasik) [reply README.md]: updated, thank you 👍

File: composer.json (hunk 2 above)
Comment (samsonasik) [reply composer.json]: updated, thank you 👍

**Comment by samsonasik** [comment]
All checks have passed 🎉 @weierophinney I think it is ready.

**Pull request merged**